orchestra/bots/staffbot.py

```diff
@@ -74,3 +82,57 @@ def __init__(self, **kwargs):
     def dispatch(self, data):
         data = self.validate(data)
         return {'text': data.get('text')}
+
+    def _send_task_to_workers(self, task, required_role):
+        # get all the workers that are certified to complete the task.
+        workers = Worker.objects.all()
```

Comment:
now that workflows/steps are in the DB, I think we can push is_worker_certified_for_task and check_worker_allowed_new_assignment into the DB as filters so we don't loop over all workers